exercise_0/main.cpp

```diff
@@ -0,0 +1,80 @@
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <vector>
+
+struct Result {
+    int vowels;
+    int consonant;
+    int others;
+};
+
+Result countLetter(std::string firstLetter) {
+    Result result;
+    std::string vowels = "AEIOUYaeouy";
+    int countVowels = 0;
+    int countConsonant = 0;
+    int countOther = 0;
+    for (int i = 0; i < firstLetter.length(); i++) {
+        if (isalpha(firstLetter[i]))
+            if (vowels.find(firstLetter[i]) != std::string::npos)
+                countVowels += 1;
+            else
+                countConsonant += 1;
+        else
+            countOther += 1;
+    }
+    result.vowels = countVowels;
+    result.consonant = countConsonant;
+    result.others = countOther;
+    return result;
+}
+
+std::string firstLetterWord(std::string inputText) {
+    std::string result = "";
+    bool v = true;
+    for (int i = 0; i < inputText.length(); i++) {
+        if (inputText[i] == ' ')
+            v = true;
+        else if (inputText[i] != ' ' && v == true) {
+            result.push_back(inputText[i]);
+            v = false;
+        }
+    }
+    return result;
```

Comment:
Are you searching for first letters of each word to create a single string with them and count then?

Reply:
yes, it does.